doc/setting_up_sidewalk_environment/setting_up_hardware.rst

```diff
@@ -88,6 +88,77 @@ Connect the Semtech shield to the Arduino-compatible headers on the development
     +---------------+-------------+
     | NRESET        | P0.04       |
     +---------------+-------------+
+   
+   .. tab:: nRF54L15 PDK
+
+      The nRF54L15 PDK does not have the Arduino-compatible header.
+      The connection layout may differ depending on the revision of your PDK.
+      GPIO assignments are recommended by the Nordic Semiconductor. 
+      However, if you need to modify the pinout, refer to the `Infocenter`_ SoC documentation, as some functions are limited to dedicated pins. 
```

Comment:
The infocenter will be shut down soon so we should refer techdocs instead (https://docs.nordicsemi.com/). Though to my knowledge there isn't any publicly available article yet that would describe it for nRF54L15 PDK. If that's the case I'm not sure if we should include this information at all

Reply:
OK removed this information. We will add a note about the GPIO limitations when the documentation for SoC will be public